Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -124,7 +124,7 @@ public final class PIFBuilder {
 
             let workspace = PIF.Workspace(
                 guid: "Workspace:\(rootPackage.path.pathString)",
-                name: rootPackage.manifestName,  // TODO: use identity instead?
+                name: rootPackage.manifest.displayName,  // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?